src/index.ts

```diff
@@ -1,15 +1,102 @@
+import { DNSoverHTTPS } from "dohdec";
 import { getLogger } from "./util/logger";
 
-const { trace } = getLogger("sayHello");
+const { trace } = getLogger("index");
+
+const DoHResolver = new DNSoverHTTPS();
+
+type RecordTypes = "openatts";
+
+type BlockchainNetwork = "ethereum";
+
+type EthereumAddress = string;
+
+enum EthereumNetworkIds {
+  homestead = 1,
+  ropsten = 3
+}
+interface OpenAttestationsDNSTextRecord {
+  type: RecordTypes;
+  net: BlockchainNetwork;
+  netId: EthereumNetworkIds;
+  address: EthereumAddress;
+  dnssec: boolean;
+}
+
+/**
+ * Returns true for strings that are openattestation records
+ * @param txtDataString e.g: '"openatts net=ethereum netId=3 address=0x0c9d5E6C766030cc6f0f49951D275Ad0701F81EC"'
```

Comment:
Do we want to separate records with space or "openatts net=ethereum,netId=3,address=0x0c9d5E6C766030cc6f0f49951D275Ad0701F81EC". Spaces are not that reliable and can be mistaken with extra spaces.

Reply:
hmm i would bring this up for discussion because personally i think spaces are a lot easier to read and it's also somewhat of a common practice for txt records - see spf

Reply:
discussed and decided to stick with spaces